lib/fog/compute/google/models/server.rb

```diff
@@ -554,7 +554,7 @@ def generate_ssh_key_metadata(username, key)
 
           ssh_keys = metadata_map["ssh-keys"] || metadata_map["sshKeys"] || ""
           ssh_keys += "\n" unless ssh_keys.empty?
-          ssh_keys += "#{username}:#{key.strip}"
+          ssh_keys += "#{username}:#{key.strip} #{username}"
```

Comment:
I guess this would be the simpliest approach if we would choose to implement the parser ourselves:
```suggestion
          ssh_keys += "#{username}:#{format_key(key, username)}"
          #....
          def format_key(key, default_comment = '')
            parts = key.strip.split
            parts << default_comment if parts.size < 3
            parts.join(' ')
          end
```